### PR TITLE
test: run integration tests using workload identity federation for GitHub actions

### PR DESCRIPTION
**Pull request by chingor13**
Uses https://cloud.google.com/blog/products/identity-security/enabling-keyless-authentication-from-github-actions

### CODE REVIEW

**Review by lesv — COMMENTED** [review]
Since this is human created and readable, shouldn't it have a license & copyright at the top?

**Review by lesv — APPROVED** [review]

**Comment by lesv** [comment]
@Neenu1995 @eaball35 You should be aware of this PR

File: .github/workflows/integration.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: integration
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+  pull_request_target:
+    types: [labeled]
+￼  schedule:
+￼  - cron:  '0 2 * * *'
```
Comment (BenWhitehead) [inline .github/workflows/integration.yaml:10]: fancy hidden characters?

**Comment by chingor13** [comment]
We're looking into how to scale this for all our repos.

File: .github/workflows/integration.yaml (hunk 1 above)
Comment (chingor13) [reply .github/workflows/integration.yaml:10]: yeah, copy/pasted, I can fix later

**Comment by alicejli** [comment]
@chingor13 any update on this?

**Pull request closed without merging**